src/vivarium_gates_mncnh/components/maternal_disorders.py

```diff
@@ -181,8 +181,35 @@ def on_time_step(self, event: Event) -> None:
         )
         pop.loc[got_disorder_idx, COLUMNS.POSTPARTUM_DEPRESSION_CASE_TYPE] = case_type
         # PPD case duration
-        pop.loc[
-            got_disorder_idx, COLUMNS.POSTPARTUM_DEPRESSION_CASE_DURATION
-        ] = self.lookup_tables["case_duration"](got_disorder_idx)
+        pop.loc[got_disorder_idx, COLUMNS.POSTPARTUM_DEPRESSION_CASE_DURATION] = (
+            self.lookup_tables["case_duration"](got_disorder_idx)
+        )
+
+        self.population_view.update(pop)
+
+
+class AbortionMiscarriageEctopicPregnancy(MaternalDisorder):
+    def __init__(self) -> None:
+        super().__init__(COLUMNS.ABORTION_MISCARRIAGE_ECTOPIC_PREGNANCY)
+
+    def load_incidence_risk(self, builder):
+        incidence_risk = super().load_incidence_risk(builder)
+        incidence_risk["value"] = 1.0
```

Comment:
Doesn't this make every value 1? If so, why do we need to use `self.incidence_risk` at all below if it's always 1?